framework/PostProcessors/MCSimporter.py

```diff
@@ -0,0 +1,196 @@
+import pandas as pd
+import numpy as np
+import csv
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from .PostProcessor import PostProcessor
+from utils import InputData, InputTypes
+from utils import xmlUtils as xmlU
+from utils import utils
+#Internal Modules End-----------------------------------------------------------
+
+
+class MCSImporter(PostProcessor):
+  """
+    This is the base class of the PostProcessor that imports Minimal Cut Sets (MCSs) into RAVEN as a PointSet
+  """
+  def __init__(self, messageHandler):
+    """
+      Constructor
+      @ In, messageHandler, MessageHandler, message handler object
+      @ Out, None
+    """
+    PostProcessor.__init__(self, messageHandler)
+    self.printTag  = 'POSTPROCESSOR MCS IMPORTER'
+    self.expand    = None  # option that controls the structure of the ET. If True, the tree is expanded so that
+                           # all possible sequences are generated. Sequence label is maintained according to the
+                           # original tree
+
+  @classmethod
+  def getInputSpecification(cls):
+    """
+      Method to get a reference to a class that specifies the input data for the class cls.
+      @ In, cls, the class for which we are retrieving the specification
+      @ Out, inputSpecification, InputData.ParameterInput, class to use for
+        specifying input of cls.
+    """
+    inputSpecification = super(MCSImporter, cls).getInputSpecification()
+    inputSpecification.addSub(InputData.parameterInputFactory("expand",       contentType=InputTypes.BoolType))
+    inputSpecification.addSub(InputData.parameterInputFactory("BElistColumn", contentType=InputTypes.StringType))
+    return inputSpecification
+
+  def initialize(self, runInfo, inputs, initDict) :
+    """
+      Method to initialize the PostProcessor
+      @ In, runInfo, dict, dictionary of run info (e.g. working dir, etc)
+      @ In, inputs, list, list of inputs
+      @ In, initDict, dict, dictionary with initialization options
+      @ Out, None
+    """
+    PostProcessor.initialize(self, runInfo, inputs, initDict)
+
+  def _localReadMoreXML(self, xmlNode):
+    """
+      Function to read the portion of the xml input that belongs to this specialized class
+      and initialize some stuff based on the inputs got
+      @ In, xmlNode, xml.etree.Element, Xml element node
+      @ Out, None
+    """
+
+    paramInput = MCSImporter.getInputSpecification()()
+    paramInput.parseNode(xmlNode)
+    self._handleInput(paramInput)
+
+  def _handleInput(self, paramInput):
+    """
+      Method that handles PostProcessor parameter input block.
+      @ In, paramInput, ParameterInput, the already parsed input.
+      @ Out, None
+    """
+
+    expand = paramInput.findFirst('expand')
+    self.expand = expand.value
+
+    if self.expand == True:
```

Comment:
```if self.expand:```

Reply:
edited!

Reply:
i also edited the same error in a different line